### PR TITLE
Fix ParticleEffect entity JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
QA not necessary because no executable code changes.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11532/HighFidelity-Beta-PR13681-1ac6845.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11532/HighFidelity-Beta-Interface-PR13681-1ac6845.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11532/HighFidelity-Beta-PR13681-1ac6845.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11532/HighFidelity-Beta-Interface-PR13681-1ac6845.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11532/HighFidelity-Beta-PR13681-1ac6845.apk

File: libraries/entities/src/EntityItemProperties.cpp [hunk 1]
```diff
@@ -886,28 +886,30 @@ EntityPropertyFlags EntityItemProperties::getChangedProperties() const {
  * @property {string} textures="" - The URL of a JPG or PNG image file to display for each particle. If you want transparency,
  *     use PNG format.
  * @property {number} particleRadius=0.025 - The radius of each particle at the middle of its life.
- * @property {number} radiusStart=NAN - The radius of each particle at the start of its life. If NAN, the
+ * @property {number} radiusStart=null - The radius of each particle at the start of its life. If <code>null</code>, the
```
Comment (SamGondelman) [inline libraries/entities/src/EntityItemProperties.cpp]: In the JS console, if I create a particle with unset start/end values, then use getEntityProperties to get it, I get NaN.

Comment (ctrlaltdavid) [reply libraries/entities/src/EntityItemProperties.cpp]: I get NaNs now, too - for radiusStart, radiusFinish, alphaStart, and  AlphaFinish. I must have been running an older version of Interface or something.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11566/HighFidelity-Beta-PR13681-5186529.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11566/HighFidelity-Beta-Interface-PR13681-5186529.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11566/HighFidelity-Beta-PR13681-5186529.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11566/HighFidelity-Beta-Interface-PR13681-5186529.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11566/HighFidelity-Beta-PR13681-5186529.apk

**Comment by huffman** [comment]
Hi @ctrlaltdavid - would you be able to fix the conflict on this one?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11685/HighFidelity-Beta-PR13681-b36b981.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11685/HighFidelity-Beta-PR13681-b36b981.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11685/HighFidelity-Beta-Interface-PR13681-b36b981.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11685/HighFidelity-Beta-PR13681-b36b981.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11685/HighFidelity-Beta-Interface-PR13681-b36b981.exe

**Pull request merged**